march_hardware/include/march_hardware/imotioncube/motor_controller.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Project March.
+
+#include "actuation_mode.h"
+#include "motor_controller_state.h"
+#include <string>
+
+namespace march
+{
+class MotorController
+{
+public:
+  virtual double getAngleRadAbsolute() = 0;
+  virtual double getAngleRadIncremental() = 0;
+  virtual double getVelocityRadAbsolute() = 0;
+  virtual double getVelocityRadIncremental() = 0;
+  virtual int16_t getTorque() = 0;
+  virtual MotorControllerStates getStates() = 0;
+
+  virtual ActuationMode getActuationMode() const = 0;
+  virtual uint16_t getSlaveIndex() const = 0;
+
+  virtual float getMotorCurrent() = 0;
+  virtual float getMotorControllerVoltage() = 0;
+  virtual float getMotorVoltage() = 0;
+  virtual bool getIncrementalMorePrecise() const = 0;
+
+  virtual void actuateRad(double target_rad) = 0;
+  virtual void actuateTorque(int16_t target_torque) = 0;
+
+  virtual void goToOperationEnabled() = 0;
```

Comment:
This method is a bit specific (for an interface), maybe rename it to `enable`, which is more general.

Reply:
Renamed it to prepareActuation, consistent with the naming in the Joint class